app/src/main/java/com/duckduckgo/app/httpsupgrade/api/HttpsUpgradeService.kt

```diff
@@ -33,4 +34,7 @@ interface HttpsUpgradeService {
 
     @GET("https://staticcdn.duckduckgo.com/https/https-mobile-bloom.bin?cache_version=1")
     fun httpsBloomFilter(): Call<ResponseBody>
+
+    @GET("https://duckduckgo.com/smarter_encryption.js")
+    fun upgradeListForPartialHost(@Query("pv1") partialMd5: String): Call<List<String>>
```

Comment:
parameter name here says `md5` but what is passed to it is `sha1`

Reply:
Good spot, we changed it.